generator/ServiceClientGeneratorLib/Generators/CodeAnalysis/CodeAnalysisProjectFile.tt

```diff
@@ -24,7 +24,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis" Version="3.0.0" />
```

Comment:
I remember we set `Microsoft.CodeAnalysis` at `3.0.0` specifically because it supports VS 2019 (this was after I broke the code analyzers last year: https://github.com/aws/aws-sdk-net/issues/2998).

Is it possible we won't support some older versions of VS with this change?

Reply:
I  just tested and this version still works in VS 2019. Probably not RTM but at some point version in VS 2019 updated to the newer version. I would like to keep using this slightly more modern version and see during preview if this causes problems for anybody. I would assume few if any are using VS 2019 RTM at this point.